Source: Ali Polatel <polatel@gmail.com>
Source: Radoslaw Szkodzinski <astralstorm@gen2.org>
Upstream: No (TODO)
Reason: Binding to 0.0.0.0 is bad, mmkay?
--- libwww-perl-5.831/t/local/chunked.t	2009-06-15 15:20:06.000000000 +0200
+++ libwww-perl-5.831/t/local/chunked.t.new	2009-09-03 18:11:13.791041011 +0200
@@ -95,7 +95,7 @@
 my $tests = @TESTS;
 my $tport = 8333;
 
-my $tsock = IO::Socket::INET->new(LocalAddr => '0.0.0.0',
+my $tsock = IO::Socket::INET->new(LocalAddr => '127.0.0.1',
                                   LocalPort => $tport,
                                   Listen    => 1,
                                   ReuseAddr => 1);
@@ -103,7 +103,7 @@
   plan skip_all => "This system cannot fork";
 }
 elsif (!$tsock) {
-  plan skip_all => "Cannot listen on 0.0.0.0:$tport";
+  plan skip_all => "Cannot listen on 127.0.0.1:$tport";
 }
 else {
   close $tsock;
@@ -158,7 +158,7 @@
 } else {
   die "cannot fork: $!" unless defined $pid;
   my $d = HTTP::Daemon->new(
-                            LocalAddr => '0.0.0.0',
+                            LocalAddr => '127.0.0.1',
                             LocalPort => $tport,
                             ReuseAddr => 1,
                            ) or die;
